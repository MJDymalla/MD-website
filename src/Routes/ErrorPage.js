import React from 'react';

const errorPage = () => {
    return (
      <div>
        <header>
          404 Page Not Found.
        </header>
      </div>
    );
  };
  
  export default errorPage;