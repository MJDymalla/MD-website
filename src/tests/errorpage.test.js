import React from 'react';
import ReactDOM from 'react-dom';
import ShallowRenderer from 'react-shallow-renderer'; 
import ErrorPage from '../Routes/ErrorPage';

test('should render header correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<ErrorPage />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
})



