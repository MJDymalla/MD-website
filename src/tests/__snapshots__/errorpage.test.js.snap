// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<div>
  <header>
    404 Page Not Found.
  </header>
</div>
`;
